data/data/aws/bootstrap/main.tf

```
@@ -154,7 +154,8 @@ resource "aws_lb_target_group_attachment" "bootstrap" {
 }
 
 resource "aws_security_group" "bootstrap" {
-  vpc_id = var.vpc_id
+  vpc_id      = var.vpc_id
+  description = "Created by OpenShift Installer"
```

I was not suggesting that you copy this string everywhere. I was suggesting that you use a [local value](https://www.terraform.io/docs/language/values/locals.html).
```
locals {
  description = "Created by OpenShift Installer"
}
```

```suggestion
  description = local.description
```

---

ah ok sorry new to terraform